distsys/20200218.md

```diff
@@ -0,0 +1,53 @@
+# 2020 February Meetup
+
+First meetup, yay!
+
+## Atendees:
+
+- [ignasi35](https://twitter.com/ignasi35): organizer
+- [@joaoalves](https://twitter.com/joaoqalves): organizer
+- [@bakwrau](https://twitter.com/bakwrau): organizer
+- [@gonzaloserrano](https://twitter.com/gonzaloserrano): organizer
+
+We met at [Enrique Tomás](https://www.google.com/maps/place/Enrique+Tom%C3%A1s+JAMON-TAPAS-VINO/@41.3857048,2.1640656,17z/data=!3m1!4b1!4m5!3m4!1s0x12a4a2f49c29d807:0xc203df08e3e94450!8m2!3d41.3857008!4d2.1662543).
+
+## Topics:
+
+Here's a list of topics we touched on in no particular order.
+
+- the definition of dist-sys
+- papers and articles
+    - [Dynamo](https://www.allthingsdistributed.com/files/amazon-dynamo-sosp2007.pdf)
+    - [Distributed Systems Are a UX Problem](https://bravenewgeek.com/distributed-systems-are-a-ux-problem/)
+- Google's SRE books ( [SRE](https://www.amazon.es/Site-Reliability-Engineering-Betsy-Beyer/dp/149192912X?SubscriptionId=AKIAILSHYYTFIVPWUY6Q&tag=duckduckgo-ffab-es-21&linkCode=xm2&camp=2025&creative=165953&creativeASIN=149192912X) and [SRE Workbook](https://www.amazon.es/Site-Reliability-Workbook-Practical-Implement/dp/1492029505?SubscriptionId=AKIAILSHYYTFIVPWUY6Q&tag=duckduckgo-ffab-es-21&linkCode=xm2&camp=2025&creative=165953&creativeASIN=1492029505))
+- [CDRTs](https://en.wikipedia.org/wiki/Conflict-free_replicated_data_type)
+- Microservices
+    - the network
+    - problem: network hops inside database transactions in your handlers
+    - microservices.io patterns
+- Unknown unkowns
+- Martin Kleppman work
+    - [Designing Data Intensive Applications](https://www.amazon.es/Designing-Data-Intensive-Applications-Reliable-Maintainable/dp/1449373321?SubscriptionId=AKIAILSHYYTFIVPWUY6Q&tag=duckduckgo-ffab-es-21&linkCode=xm2&camp=2025&creative=165953&creativeASIN=1449373321) book
+    - https://github.com/automerge/automerge
+- https://github.com/dapr/dapr
```

Comment:
@gonzaloserrano @ignasi35 you also referenced some work around clocks. Do you think it would be useful to add that here?

Reply:
yep!